doc/docs/master/how-tos/export-data-out-pachyderm/mount-repo-to-local-computer.md

```diff
@@ -0,0 +1,187 @@
+
+  !!! note
+      macOS has limited support for FUSE and might not be stable.
+
+## Mount a Pachyderm Repo
+
+Before you can mount a Pachyderm repo, verify that you have all the
+[Prerequisites](#prerequisites).
+
+To mount a Pachyderm repo on a local computer, complete the following
+steps:
+
+1. In a terminal, go to a directory in which you want your
+Pachyderm repo. It can be any directory on your local computer,
+except for the `~/` directory. Mounting to the root directory will
+fail.
+
+1. Run `pachctl mount` for a repository and branch that you want to mount:
+
+   ```bash
+   pachctl mount <path-on-your-computer> [flags]
+   ```
+
+   **Example:**
+
+   * If you want to mount all the repositories in your Pachyderm cluster 
+   to a `pfs` directory on your computer and give `WRITE` access to them, run:
+
+   ```bash
+   pachctl mount pfs --write
+   ```
+
+   * If you want to mount the master branch of the `images` repo
+   and enable file editing in this repository, run:
+
+   ```bash
+   pachctl mount images --repos images@master+w
+   ```
+
+   To give read-only access, omit `+w`.
+
+   **System Response:**
+
+   ```bash
+   ro for images: &{Branch:master Write:true}
+   ri: repo:<name:"montage" > created:<seconds:1591812554 nanos:348079652 > size_bytes:1345398 description:"Output repo for pipeline montage." branches:<repo:<name:"montage" > name:"master" >
+   continue
+   ri: repo:<name:"edges" > created:<seconds:1591812554 nanos:201592492 > size_bytes:136795 description:"Output repo for pipeline edges." branches:<repo:<name:"edges" > name:"master" >
+   continue
+   ri: repo:<name:"images" > created:<seconds:1591812554 nanos:28450609 > size_bytes:244068 branches:<repo:<name:"images" > name:"master" >
+   MkdirAll /var/folders/jl/mm3wrxqd75l9r1_d0zktphdw0000gn/T/pfs201409498/images
+   ```
+
+   The command runs in your terminal until you terminate it
+   by pressing `CTRL+C`.
+
+1. You can check that the repo was mounted by running the mount command
+in your terminal:
+
+   ```bash hl_lines="7"
+   mount
+   /dev/disk1s1 on / (apfs, local, read-only, journaled)
+   devfs on /dev (devfs, local, nobrowse)
+   /dev/disk1s2 on /System/Volumes/Data (apfs, local, journaled, nobrowse)
+   /dev/disk1s5 on /private/var/vm (apfs, local, journaled, nobrowse)
+   map auto_home on /System/Volumes/Data/home (autofs, automounted, nobrowse)
+   pachctl@osxfuse0 on /Users/testuser/pachyderm/images (osxfuse, nodev, nosuid, synchronous, mounted by testuser)
+   ```
+
+1. Access your mountpoint.
+
+   For example, in macOS, open Finder, press
+   `CMD + SHIFT + G`, and type the mountpoint location. If you have mounted
+   the repo to `~/pachyderm/images`, type `~/pachyderm/images`.
+
+   ![finder-repo-mount](../../assets/images/s_finder_repo_mount.png)
+
+1. Edit the files as needed.
+1. When ready, add your changes to the Pachyderm repo by stopping
+the `pachctl mount` command with `CTRL+C`.
```

Comment:
As noted in other comments -- does unmount work here? Is ti reliable on OSX?

Reply:
it is not reliable on macOS. Added pachctl unmount for reference.